### ISSUE TITLE
Exiting image from Block quote throws an error

### ISSUE THREAD

**Issue by przemyslaw-zan** [ckeditor/ckeditor5#10266]
## 📝 Provide detailed reproduction steps (if any)

1.  Go to Examples > Document Editor
2.  Select an image
3.  Click `Block quote` button twice

### ✔️ Expected result

The image goes in and out of quote block

### ❌ Actual result

On second click, error is thrown: `Uncaught CKEditorError: t.mapper.toModelElement(...) is undefined`

## 📃 Other details

*   Browser: Tested on FF, Chrome and Edge
*   OS: Tested on Windows

**Comment by FilipTokarski** [comment]
Stack trace from manual test:
```js
Uncaught TypeError: Cannot read property 'root' of undefined
    at Document.<anonymous> (widgetresize.js:99)
    at Document.fire (emittermixin.js:199)
    at Document._handleChangeBlock (document.js:309)
    at Model._runPendingChanges (model.js:842)
    at Model.change (model.js:184)
    at BlockQuoteCommand.execute (blockquotecommand.js:54)
    at BlockQuoteCommand.<anonymous> (observablemixin.js:258)
    at BlockQuoteCommand.fire (emittermixin.js:199)
    at BlockQuoteCommand.<computed> [as execute] (observablemixin.js:262)
    at CommandCollection.execute (commandcollection.js:69)
```

**Comment by maxbarnas** [comment]
The same error will show up in Decoupled Editor sample (http://localhost:8080/ckeditor5-build-decoupled-document/sample/) when you:
1. Make the image `inline`.
2. Make the image a numbered/bulleted list item.
3. Click "Decrease indent".

**Comment by FilipTokarski** [comment]
Bisection shows this commit: https://github.com/ckeditor/ckeditor5/commit/0bf7ba6d590b5f10e72df2ae84aef49e724f996d

**Comment by oleq** [comment]
Closed in #10280.
